rootfs/chroot.sh

```diff
@@ -41,7 +43,8 @@ for dir in "${writeDirs[@]}"; do
 done
 
 mkdir -p  /chroot/lib /chroot/proc /chroot/usr /chroot/bin /chroot/dev /chroot/run 
-cp /etc/passwd /etc/group /chroot/etc/
+cp /etc/passwd /etc/group /etc/hosts /chroot/etc/
```

Comment:
Question 🙋‍♂️ why we need copy hosts?

Reply:
chroot and mirror test were failing due to it 🤷‍♂️

Apparently something has changed between versions but I couldn't figure out. NGINX uses "localhost" to mirror things, but it cannot resolve "localhost" because /etc/hosts is not inside the chroot